src/Clockify/EntityExtractorUtil.cs

```diff
@@ -16,8 +15,7 @@ public static string GetWorkedEntity(TimeSurveyBotLuis._Entities._Instance entit
                 workedEntityInstances.First().Text == null
             )
             {
-                throw new InvalidWorkedEntityException(
-                    "I can see you want to report some hours, but I really can't understand on what 😕");
+                throw new InvalidWorkedEntityException("No worked entity has been recognized");
```

Comment:
why string here?

Reply:
This string is not returned to the user. The exception is handled in the upper layer